alpha/alpha-proc/src/main/java/org/neo4j/graphalgo/influenceΜaximization/CELFProc.java

```diff
@@ -0,0 +1,91 @@
+package org.neo4j.graphalgo.influenceΜaximization;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+import org.neo4j.graphalgo.AlgoBaseProc;
+import org.neo4j.graphalgo.AlgorithmFactory;
+import org.neo4j.graphalgo.AlphaAlgorithmFactory;
+import org.neo4j.graphalgo.api.Graph;
+import org.neo4j.graphalgo.config.GraphCreateConfig;
+import org.neo4j.graphalgo.core.CypherMapWrapper;
+import org.neo4j.graphalgo.core.concurrency.Pools;
+import org.neo4j.graphalgo.impl.influenceMaximization.CELF;
+import org.neo4j.graphalgo.result.AbstractResultBuilder;
+import org.neo4j.graphalgo.results.InfluenceMaximizationResult;
+import org.neo4j.procedure.Description;
+import org.neo4j.procedure.Name;
+import org.neo4j.procedure.Procedure;
+
+import static org.neo4j.procedure.Mode.READ;
+
+public class CELFProc extends AlgoBaseProc<CELF,CELF,InfluenceMaximizationConfig>
+{
+    private static final String DESCRIPTION =
+            "CELF algorithm for Influence Maximization aims to find `k` nodes that maximize the expected spread of influence in the network";
+
+    @Procedure( name = "gds.alpha.celf.stream", mode = READ )
```

Comment:
I would prefer if we called the procedures gds.alpha.influenceMaximization.celf.stream etc. This is very verbose but would follow our general pattern more.